feconf.py

```diff
@@ -169,6 +169,16 @@ def get_empty_ratings():
 # Empty scaled average rating as a float.
 EMPTY_SCALED_AVERAGE_RATING = 0.0
 
+# To use GAE email service.
+GAE_EMAIL_SERVICE = 'gae_email_service'
+# To use mailgun email service.
+MAILGUN_EMAIL_SERVICE = 'mailgun_email_service'
+# Use GAE email service by default.
+EMAIL_SERVICE_PROVIDER = GAE_EMAIL_SERVICE
+# Mailgun api key should be replaced with None below.
```

Comment:
If the Mailgun email API is used, the "None" below should be replaced with the Mailgun API key.